src/python/pants/backend/jvm/targets/jar_library.py

```diff
@@ -77,7 +77,7 @@ def excludes(self):
     return self.payload.excludes
 
   @property
-  def exports(self):
+  def exports_targets(self):
```

Comment:
Hm. It might be ok. If we can be pretty sure no one is using it, and won't adopt it in 1.3.0 then we could do it. Actually, if we merged the update into 1.3.0 before the release, then it's moot because it was never part of a non-dev release.